Makefile.am

```diff
@@ -34,3 +34,10 @@ tests/run-test-suite:
 	-git branch --track run-test-suite origin/run-test-suite
 	-git worktree prune
 	git worktree add $@ run-test-suite
+
+packaging:
+	git branch --track packaging origin/packaging
+	git worktree add packaging packaging
```

Comment:
`git worktree add --force $@ $@`

This command can sometimes fail without `--force`.

Reply:
done